app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,18 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          angular.forEach(scope.pod.spec.volumes, function(volumeData) {
+            if (volumeData.persistentVolumeClaim !== undefined) {
```

Comment:
for style consistency
```
if (volumeData.persistentVolumeClaim)
```
there's no reason to distinguish between null and undefined which are both falsey, if the persistentVolumeClaim sub-object exists then that if check will pass.  The only time we specifically check for null/undefined is when we need to distinguish between non-existence and zero integer values.  In those cases we have an isNil filter that handles both null and undefined.